frontend/assets/stylesheets/base/_tables.scss

```diff
@@ -44,16 +44,13 @@ td {
         border-bottom: 1px dotted $c-neutral8;
     }
 
-}
-
-.table--responsive-font {
     @include mq($from: desktop) {
         th,
         td,
         thead td {
             font-size: rem(14px);
```

Comment:
Theres probable a font-size mixin for this, so it comes from the font-size scale